### PR TITLE
Enhancement: We can now set auth.server.prefix and auth.server.suffix…

### PR DESCRIPTION
**Pull request by KevinFHartmann**
… to set the

scope prefix and suffix that we require for write access (ie. prefix: id, suffix: WRITE)
Enhancement: Our security configuration now works with values that are not
hard-coded at compile-time.

### CODE REVIEW

**Comment by KevinFHartmann** [comment]
ok to test

File: dcc-id-server/src/main/conf/application.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@
 spring:
   profiles.active: production
 
+auth:
+  server:
+    prefix: id
+    suffix: create.WRITE
```
Comment (andricDu) [inline dcc-id-server/src/main/conf/application.yml]: why don't we just have the suffix as `WRITE`

so that the correct scope would be `id.WRITE`

**Review by andricDu — CHANGES_REQUESTED** [review]

File: dcc-id-server/src/main/conf/application.yml (hunk 1 above)
Comment (KevinFHartmann) [reply dcc-id-server/src/main/conf/application.yml]: That makes sense. :-)

**Comment by KevinFHartmann** [comment]
ok to test

**Comment by KevinFHartmann** [comment]
We will need this version of the id server to work with ego.

**Review by andricDu — APPROVED** [review]

**Review by rtisma — APPROVED** [review]

**Pull request merged**